### PR TITLE
doc updates (docker support)

### PR DESCRIPTION
**Pull request by edublancas**
updates docs to show how to deploy flask and docker-based apps

----
:books: Documentation preview :books:: https://ploomber-doc--13.org.readthedocs.build/en/13/

### CODE REVIEW

File: doc/apps/docker.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# assumes app.py contains your flask app
+COPY app.py app.py
+# install flask and gunicorn
+RUN pip install flask gunicorn
+
+# this configuration is needed for your app to work, do not change it
+ENTRYPOINT ["gunicorn", "app:app", "run", "--bind", "0.0.0.0:80", "--env", "SCRIPT_NAME=/{{PROJECT_ID}}"]
+```
+
+Once you have all your files, create a zip file.
```
Comment (neelasha23) [inline doc/apps/docker.md:30]: A sample folder structure might be helpful

File: doc/apps/flask.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Flask
+
+To deploy a Flask application in Ploomber Cloud you need:
+
+- A `Dockerfile`
+- Your code
+## `Dockerfile`
+
+You need to provide a `Dockerfile`, you can use this [template](https://github.com/ploomber/doc/blob/main/examples/docker/flask/Dockerfile) to get started. The template contains the minimal steps needed for a deployment but you need to modify so it installs any required dependencies and copies your code into the Docker image.
```
Comment (neelasha23) [inline doc/apps/flask.md:9]: This page looks similar to the Docker one?

File: doc/quickstart/app.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## 1. Click on "Dashboards" -> "New"
+
+![](../static/dashboards-new.png)
+
+
+## 2. Fill out the form
+
+
+![](../static/dashboards-form.png)
```
Comment (neelasha23) [inline doc/quickstart/app.md:11]: This image displays `Ensure Voila is selected`. I think user should be able to select any of the options?

File: doc/quickstart/app.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## 3. Wait for the deployment to finish
+
+You'll see two tabs. First, you'll see the `DOCKER LOGS`, which show the progress of installing your dashboards's dependencies:
+
+![](../static/dashboards-logs.png)
+
+You can track progress progress completion from the status bar:
```
Comment (neelasha23) [inline doc/quickstart/app.md:19]: `progress` appears twice

File: doc/apps/docker.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- A `Dockerfile`
+- Your code
+## `Dockerfile`
+
+You need to provide a `Dockerfile`, you can use this [flask-based template](https://github.com/ploomber/doc/blob/main/examples/docker/flask/Dockerfile) to get started.
+
+For a successful deployment, you need the following:
+
+- Your app must run in port 80
+- You'll be assigned a  `ploomberapp.io/{{PROJECT_ID}}` URL, hence, your app must accept requests from the `/{{PROJECT_ID}}` path.
```
[the remark this answers is not in the document]
Comment (edublancas) [reply doc/apps/docker.md]: https://github.com/ploomber/doc/pull/13/commits/d2c119f1313128f47b6e246444f146a05e8444b3

File: doc/apps/docker.md (hunk 1 above)
Comment (edublancas) [reply doc/apps/docker.md]: https://github.com/ploomber/doc/pull/13/commits/c2235c9a6ce5077a0393a07a479aa7e10b240816

File: doc/apps/flask.md (hunk 2 above)
Comment (edublancas) [reply doc/apps/flask.md:9]: yeah, that's on purpose. for the docker one we have to choose a framework and currently, I've only got flask working

File: doc/quickstart/app.md (hunk 3 above)
Comment (edublancas) [reply doc/quickstart/app.md:11]: streamlit and shiny don't work (they display a banner saying "streamlit/shiny is coming soon"). 

for docker, we don't have many examples yet, so voila is the one we chose for showing how to deploy something quickly

File: doc/quickstart/app.md (hunk 4 above)
Comment (edublancas) [reply doc/quickstart/app.md]: https://github.com/ploomber/doc/pull/13/commits/bac1e2adfb327932c562f5ef8489c53b418ce069

**Review by neelasha23 — APPROVED** [review]

**Comment by neelasha23** [comment]
I do not have `Resolve conversation` permission on this project @edublancas

**Comment by edublancas** [comment]
@neelasha23 thanks for letting me know, I think you should've permission now

**Pull request merged**